CHANGELOG.rst

```diff
@@ -14,6 +14,8 @@ adheres to `Semantic Versioning <http://semver.org/spec/v2.0.0.html>`_.
 Added
 ~~~~~
 
+- Added single channel noise reduction sub-package ``scnr`` with spectral subtraction
```

Comment:
I'm wondering if we can find a more descriptive name than `scnr`. Let me throw in a few proposals that you can feel free to extent or ignore:
* denoise
* enhance
* nr (noise removal)
* nzrm (NoiZe ReMoval, pretty bad I know)
* ...

Reply:
"nzrm" 🤣 I think "denoise" is good. it's the ultimate goal and concise 👍